### DISCUSSION TITLE
Model Zoo Requests

### DISCUSSION THREAD

**Discussion by tjake** [tjake/Jlama#37] [General]
Please request quantized models you would like to see added to https://huggingface.co/tjake here.

**Comment by vaiju1981** [comment]
Hi Jake,

can you upload quantized safe tensors  for the following 3 models.
1. https://huggingface.co/NousResearch/Yarn-Mistral-7b-128k/tree/main ( this pytorch model and not safetensors)
2. https://huggingface.co/tdh87/Yarn-Solar-10b-64k-SafeTensors/tree/main
3. https://huggingface.co/microsoft/Phi-3-mini-128k-instruct/tree/main

**Comment by DumiJDev** [comment]
Hi, @tjake,

Can you quantized safetensors for the following model:
- https://huggingface.co/AstroMLab/AstroSage-8B-GGUF/blob/main/AstroSage-8B-Q8_0.gguf
